pkg/scheduler/framework/runtime/framework.go

```diff
@@ -1016,62 +1016,56 @@ func (f *frameworkImpl) RunScorePlugins(ctx context.Context, state *framework.Cy
 	allNodePluginScores := make([]framework.NodePluginScores, len(nodes))
 	numPlugins := len(f.scorePlugins) - state.SkipScorePlugins.Len()
 	plugins := make([]framework.ScorePlugin, 0, numPlugins)
-	pluginToNodeScores := make(map[string]framework.NodeScoreList, numPlugins)
+	pluginToNodeScores := make([]framework.NodeScoreList, numPlugins)
```

Comment:
need to change it to slice because we encountered `concurrent map read and map write` if it was map